packages/components/src/utils/select/keyboard-behavior.ts

```diff
@@ -0,0 +1,249 @@
+import { forceUpdate } from '@stencil/core';
+import { HTMLStencilElement } from '@stencil/core/internal';
+
+export const selectActions = {
```

Comment:
I guess this was the `enum` before and this is where the numeric values are coming from. What's the benefit? Or why not use define a literal type directly instead?